src/DotLiquid.Tests/Filters/StandardFiltersV20Tests.cs

```diff
@@ -0,0 +1,104 @@
+using System;
+using System.Collections;
+using System.Globalization;
+using NUnit.Framework;
+
+namespace DotLiquid.Tests.Filters
+{
+    [TestFixture]
+    public class StandardFiltersV20Tests : StandardFiltersTestsBase
+    {
+        public override SyntaxCompatibility SyntaxCompatibilityLevel => SyntaxCompatibility.DotLiquid20;
+        public override CapitalizeDelegate Capitalize => i => LegacyFilters.Capitalize(_context, i);
```

Comment:
nit: avoid single letter character variable names. Consider using full English terms.

Reply:
Not going to fix this, it is in test suite within a lambda expression to proxy parameter 1, 2, 3, n. Using a single letter is acceptable in this case for easier readability of the critical part here - LEGACYFILTERS vs STANDARDFILTERS.